src/creature.cpp

```diff
@@ -229,15 +229,19 @@ void Creature::onWalk()
 
 void Creature::onWalk(Direction& dir)
 {
-	if (hasCondition(CONDITION_DRUNK)) {
-		uint32_t r = uniform_random(0, 20);
-		if (r <= DIRECTION_DIAGONAL_MASK) {
-			if (r < DIRECTION_DIAGONAL_MASK) {
-				dir = static_cast<Direction>(r);
-			}
-			g_game.internalCreatureSay(this, TALKTYPE_MONSTER_SAY, "Hicks!", false);
-		}
+	uint32_t rand;
+
+	if (!hasCondition(CONDITION_DRUNK)) {
+		return;
 	}
+
+	rand = uniform_random(0, 99);
```

Comment:
uint32_t rand = 
and delete the variable above. Honestly I don't care if its in our coding styl guideline or not, but it just seems wrong, there is no point to explicitly define it like that, if we know the value at the time of declaration

Reply:
Sure. I just pushed the changes. Let me know if there is anything else that doesn't meet the coding style criteria.